crates/storage/src/transactional.rs

```diff
@@ -33,23 +33,29 @@ use crate::{
     kv_store::KVItem,
 };
 
-/// Provides a view of the storage at the given height.
+/// Provides a view of the storage at the latest height.
 /// It guarantees to be atomic, meaning the view is immutable to outside modifications.
 pub trait AtomicView: Send + Sync {
```

Comment:
I find the wording/naming to be unclear. How can it provide a view of the latest height, but then not change when the latest height changes?

Reply:
I also decided to move the `latest_height` to `HistoricalView` since it looks like a better place for this logic.